Assets/HoloToolkit/Utilities/Scripts/Editor/SolverInBetweenEditor.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.﻿
+
+using UnityEditor;
+
+namespace HoloToolkit.Unity.UX
+{
+    [CustomEditor(typeof(SolverInBetween))]
+    public class SolverInBetweenEditor : Editor
+    {
+        private SerializedProperty trackedObjectReferenceProperty;
+        private SerializedProperty transformTargetProperty;
+        private SolverInBetween solverInBetween;
+
+        private static readonly string[] fieldsToExclude = new string[] { "m_Script", "secondTransformOverride" };
+
+        private void OnEnable()
+        {
+
+            trackedObjectReferenceProperty = serializedObject.FindProperty("trackedObjectForSecondTransform");
+            transformTargetProperty = serializedObject.FindProperty("secondTransformOverride");
+
+            solverInBetween = target as SolverInBetween;
+        }                    
+
+        public override void OnInspectorGUI()
+        {
+            serializedObject.Update();
+
+            EditorGUILayout.Space();
+
+            EditorGUI.BeginChangeCheck();
+            EditorGUILayout.PropertyField(trackedObjectReferenceProperty);
+            bool trackedObjectChanged = EditorGUI.EndChangeCheck();
+
+            EditorGUILayout.PropertyField(transformTargetProperty);
+
+            DrawPropertiesExcluding(serializedObject, fieldsToExclude);
```

Comment:
What's this for?

Reply:
DrawPropertiesExcluding() is an easy way to fallback to the default inspector display while excluding the elements you've created custom behavior for. Here I'm excluding m_Script (since it would otherwise show up half way down the inspector pane) and the secondTransformOverride, which is already settable via transformTargetProperty.

Reply:
Why not just add the HideInInspector attribute?

Reply:
You can't apply [HideInInspector] to m_Script because it's not a variable in SolverInBetween.

Reply:
Idk, this just seems a bit too hacky. I bet there's a better way to fix this.

Reply:
`DrawPropertiesExcluding` is a built-in Unity method, so it seems acceptable to me.

![image](https://user-images.githubusercontent.com/3580640/41552062-87048438-72e2-11e8-832f-cb2c02bb38dd.png)

Other options would be:
1. Having the script field appear in the middle of the inspector
![image](https://user-images.githubusercontent.com/3580640/41553543-e74ee226-72e6-11e8-903d-1ec4cc2cb636.png)
2. Rebuilding the entire inspector from scratch

Neither of which are better alternatives, in my opinion.

Reply:
I agree with Kurtis, I don't think there's a more elegant way of redoing this. The DrawPropertiesExcluding pattern is much easier to extend and maintain than a full rewrite. If anything, we should encourage other full inspector rewrites to use the DrawPropertiesExcluding pattern instead so it becomes more recognizable.